src/frontend/src/pages/CSSAPIPage.tsx

```diff
@@ -283,6 +286,80 @@ const CSSAPIPage = () => {
               </Stack>
             </Card>
           </Stack>
+          <Stack direction="row">
+            {/* GET USER */}
+            <Card paddingY="10px">
+              <Stack>
+                <Stack direction="row">
+                  <Txt bold>getKCUser</Txt>
+                  <Txt>Get user details.</Txt>
+                </Stack>
+                <hr />
+                <Stack direction="row" center>
+                  <input
+                    type="text"
+                    placeholder="Type a user's first name"
+                    onChange={(e) => setGetUserInput(e.target.value)}
+                  ></input>
+                  <Button
+                    size="s"
+                    onClick={() => {
+                      if (getUserInput !== "")
+                        callAPI(
+                          "/cssAPI/getKCUser",
+                          "GET",
+                          `?user=${getUserInput}`
+                        );
```

Comment:
I think we should abstract the onclicks out as a function and pass that back in here